### PR TITLE
Gordeenko/Task4 - in work

### PR DESCRIPTION
**Pull request by IrinaGordeenko**
Task4 - in work

### CODE REVIEW

**Review by MikitaTsiarentsyeu — CHANGES_REQUESTED** [review]
OK, I will wait for a complete version)

**Comment by IrinaGordeenko** [comment]
> OK, I will wait for a complete version)

Add new file - Task4_new.py
Code Update for Task4, but I have a question.
How to create a list from the terminal? How to do it from parallel (print) ?

File: Tasks/Gordeenko/Task4/Task4_new.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            z += 1
+            print(i, end ="")
+            if (z>= user_input) and (i == " "):
+                z = 0
+                print('\r')
+
+
+with open (os.path.join(os.getcwd(), "Task4", "text_new.txt"), 'w', encoding="utf8") as f:
+    f.seek(0)
+    f.writelines(l)
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Gordeenko/Task4/Task4_new.py:32]: l is the last line of the file, not exactly correct

**Review by MikitaTsiarentsyeu — APPROVED** [review]
This could be better, the score is 7.

As for your question - lets discuss it in class, it will be more clear.

**Pull request merged**